src/drive/containers/File.jsx

```diff
@@ -53,7 +55,12 @@ export const getClassFromMime = attrs => {
   }
 
   return styles[
-    'fil-file-' + getTypeFromMimeType(styles, 'fil-file-')(attrs.mime)
+    'fil-file-' +
+      (getTypeFromMimeType(styles, 'fil-file-')(attrs.mime) ||
+        console.warn(
```

Comment:
Clever trick, maybe too clever? (but still fine!)

Reply:
Not easy to read for a new comer, but this code is not central and it worth be concise.